src/main.ts

```diff
@@ -94,6 +96,10 @@ export class ImportContext {
 				this.importedCountEl = el.createDiv({ cls: 'importer-stat-count', text: this.notes.toString() });
 				el.createDiv({ cls: 'importer-stat-name', text: 'imported' });
 			});
+			el.createDiv('importer-stat mod-updated', el => {
```

Comment:
I'm not sure yet that we should add this, since only one importer would be using it. I think we should continue using the regular imported count for now.